src/main/java/com/example/authservice/entity/User.java

```diff
@@ -14,9 +15,11 @@ public class User {
     @GeneratedValue(strategy = GenerationType.AUTO)
     private Long id;
 
-    @Column(nullable = false, unique = true)
+    @NotBlank(message = "Email is a required field")
```

Comment:
```suggestion
    @NotBlank(message = "is a required field")
```
Error message shows a repetitive "_email Email is a required field_" and we could resolve this by removing the name of the field in the error message

![Screenshot from 2022-05-13 14-23-04](https://user-images.githubusercontent.com/55695513/168282487-eef79b36-2055-4313-9167-f14f3a98aab4.png)